internal/cel/cel.go

```diff
@@ -67,9 +68,19 @@ func NewCELParser() *CELParser {
 		checksum,
 	)
 
+	eventEnv, _ := cel.NewEnv(
+		cel.Declarations(
+			decls.NewVar("input", decls.NewMapType(decls.String, decls.Dyn)),
+			decls.NewVar("additional_metadata", decls.NewMapType(decls.String, decls.Dyn)),
+			decls.NewVar("event_id", decls.NewMapType(decls.String, decls.Dyn)),
```

Comment:
Shouldn't this be a string not a map? (might have to fix the `workflow_run_id` as well if this was copied from that)